oxd-server/src/test/java/org/gluu/oxd/server/GetAuthorizationUrlTest.java

```diff
@@ -49,6 +52,24 @@ public void testWithParameterAuthorizationUrl(String host, String redirectUrl, S
         assertNotNull(resp);
         notEmpty(resp.getAuthorizationUrl());
         assertTrue(resp.getAuthorizationUrl().contains(paramRedirectUrl));
+    }
+
+    @Parameters({"host", "redirectUrl", "opHost"})
+    @Test
+    public void testWithParams(String host, String redirectUrl, String opHost) {
+        ClientInterface client = Tester.newClient(host);
+
+        final RegisterSiteResponse site = RegisterSiteTest.registerSite(client, opHost, redirectUrl);
+        final GetAuthorizationUrlParams commandParams = new GetAuthorizationUrlParams();
+        commandParams.setOxdId(site.getOxdId());
+
+        Map<String, String> params = new HashMap<>();
+        params.put("max_age", "70");
+        params.put("is_valid", "true");
+        commandParams.setParams(params);
 
+        final GetAuthorizationUrlResponse resp = client.getAuthorizationUrl(Tester.getAuthorization(), commandParams);
+        assertNotNull(resp);
+        notEmpty(resp.getAuthorizationUrl());
```

Comment:
add assert on `&max_age=70` and `is_valid=true`

Reply:
Change done